### PR TITLE
fix(docs/guide/component): wrong $componentController usage

### PR DESCRIPTION
**Pull request by ptgamr**
#14091

### CODE REVIEW

**Comment by ptgamr** [comment]
I signed it!

File: docs/content/guide/component.ngdoc [hunk 1]
```diff
@@ -420,18 +420,19 @@ The examples use the [Jasmine](http://jasmine.github.io/) testing framework.
...
 ```js
 describe('component: heroDetail', function() {
-  var component, scope, hero;
+  var component, scope, hero, componentController;
 
   beforeEach(module('simpleComponent'));
 
   beforeEach(inject(function($rootScope, $componentController) {
     scope = $rootScope.$new();
+    componentController = $componentController;
```
Comment (gkalpak) [inline docs/content/guide/component.ngdoc]: It is better to keep the original name of the service.
Name the variable `$componentController` and inject it as `_$componentController_`.

**Comment by gkalpak** [comment]
@ptgamr, thx ! Could you, please,  address my comment and ping me ?

**Comment by gkalpak** [comment]
@ptgamr, I see you tried to use the correct commit message format :+1: 
If you want to go the extra mile, you can change it to something like:

```text
docs(guide/component): fix `$componentController` usage

Fixes #14091
```

<sub>(Adding `Fixes/Closes #XYZ` at the end, closes the related issues automatically when merging the PR.)</sub>

**Comment by ptgamr** [comment]
@gkalpak I've updated the PR. Thanks for your comment!

**Comment by gkalpak** [comment]
Backported to `v1.5.x` as 2dda75b.
Thx @ptgamr !

**Comment by ptgamr** [comment]
Thanks @gkalpak  :+1:

**Pull request closed without merging**